web/src/games/barnbarter/helpers.ts

```diff
@@ -0,0 +1,82 @@
+import { IG } from './definitions';
+import { Ctx } from 'boardgame.io';
+
+export function finished(G: IG, ctx: Ctx) {
+  if (G.cards.length !== 0) {
+    return false;
+  }
+
+  for (var i = 0; i < ctx.numPlayers; i++) {
+    var animals = G.players[i].cards.map((card) => {
```

Comment:
Use let instead of var, here and eveywhre else. The scope of Var is global and we don't want that

Reply:
yup i'm terrible at typescript. Thanks for the hint :)